project-base/tests/FrontendApiBundle/Functional/Image/ProductImagesTest.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Image;
+
+use Shopsys\FrameworkBundle\Model\Product\ProductFacade;
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class ProductImagesTest extends GraphQlTestCase
+{
+    /**
+     * @var \Shopsys\ShopBundle\Model\Product\Product
+     */
+    private $product;
+
+    protected function setUp(): void
+    {
+        $productFacade = $this->getContainer()->get(ProductFacade::class);
+        $this->product = $productFacade->getById(1);
+
+        parent::setUp();
```

Comment:
Should not be parent's setup called at first place and then overide it with getting `ProductFacade` and `Product`? When you edit parent's setup you might change expected behavior here.

Reply:
fixed